.gitignore

```diff
@@ -9,7 +9,6 @@ oauth/user.json
 .idea
 *.iml
 *.xcf
-example.eskip
```

Comment:
maybe someone has always an example.eskip here

Reply:
we'll see :)